runelite-client/src/main/java/net/runelite/client/plugins/regenmeter/RegenMeterConfig.java

```diff
@@ -43,7 +45,9 @@ default boolean showHitpoints()
 	@ConfigItem(
 		keyName = "showSpecial",
 		name = "Show Spec. Attack regen",
-		description = "Show a ring around the Special Attack orb")
+		description = "Show a ring around the Special Attack orb",
+		position = 1
```

Comment:
Why 1 after 2? Swap the order?
Don't take my idea as final though, a maintainer would know more than I do on this topic.

Reply:
The plugin config would sort it based on position so the position here doesn't matter but people order them here anyway.